addons/sourcemod/translations/ru/shop.phrases.txt

```diff
@@ -448,6 +448,22 @@
 		"ru"		"{green}[Shop] {default}Ваши данные загружаются!"
 	}
 
+	"You Buy Sure"
+	{
+		"ru"		"Вы уверены что хотите купить этот предмет?"
+	}
+
+	"You Sell Sure"
+	{
+		"ru"		"Вы уверены что хотите продать этот предмет?"
+	}
+
+	"confirm_luck"
+	{
+		"#format"	"{1:i}"

```

Comment:
it is needed only in main translation file. 
I forgot to remove this line when I asked for a translation

Reply:
did you mean remove "#format"	"{1:i}"?